README.rst

```diff
@@ -40,6 +40,18 @@ UCI configuration options must go in ``/etc/config/monitoring``.
 - ``monitored_interfaces``: interfaces that needs to be monitored, defaults to ``*`` for all interfaces.
 - ``interval``: time after which device data should be sent to server, defaults to ``300``.
 - ``verbose_mode``: can be enabled (set to ``1``) to ease `debugging <#debugging>`__ in case of issues, defaults to ``0`` (disabled).
+- ``required_memory``: available memory required to save data temporarily, defaults to ``0.05``.
+- ``max_retries``: number of time the service should try to send data to server in case of failure, defaults to ``5``.
```

Comment:
suggestion:

maximum number of retries in case of failures to send data to the server, defaults to ``5``.
in case the maximum number of retries is reached .... (link the section below so the user knows what happens in this case).